MarvelApiClient/CharactersApiClient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharactersApiClient.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente on 14/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BrightFutures
+
+public class CharactersApiClient : BaseApiClient {
+
+    private let parser: CharactersParser
+
+    init(baseEndpoint: String, timeProvider: TimeProvider, httpClient: HttpClient, parser: CharactersParser) {
+        self.parser = parser
+        super.init(baseEndpoint: baseEndpoint, timeProvider: timeProvider, httpClient: httpClient)
```

Comment:
Wouldn't it be better to have BaseApiClient as a property instead of subclassing ?
If the client use it with dependency injection it could reuse the same instance accros [Chararactes|Comics|Authros]ApiClient.